cljfmt/test/cljfmt/test_util/cljs.cljc

```diff
@@ -1,5 +1,5 @@
 (ns cljfmt.test-util.cljs
-  (:require [cljs.test :as test]
+  (:require [clojure.test :as test]
```

Comment:
This change was needed since bb loads this namespace and `cljs.test` isn't part of `bb`. Loading `clojure.test` also works in ClojureScript so this change doesn't change any behavior.